### PR TITLE
Add links and instructions on running with Quantum Mobile images

### PR DESCRIPTION
**Pull request by unkcpz**
Link and instructions for:
- M1/M2 apple silicon
- x86_64

### CODE REVIEW

File: docs/source/installation/access_aiidalab/vm.rst [hunk 1]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
 
 .. note::
 
-   If you used another method to access AiiDAlab, you may proceed to :doc:`install the app </installation/install>`.
+   If you uses another method to access AiiDAlab, you may proceed to :doc:`install the app </installation/install>`.
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:7]: "used" is correct. Also, note that this note appears in several places in the site. We need to be consistent.

File: docs/source/installation/access_aiidalab/vm.rst [hunk 2]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
 
-The AiiDAlab virtual image based on `Quantum Mobile <https://quantum-mobile.readthedocs.io/>`_ is available for the following architectures:
+The AiiDAlab Quantum ESPRESSO virtual image based on `Quantum Mobile <https://quantum-mobile.readthedocs.io/>`_ is available for the following architectures:
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:9]: When using "the following...:", ideally you should follow with a list. I recommend you add a bullet-point list with just the names, then add "See the following sections for instructions pertaining to your architecture." or something like it.

Also, maybe break down the sentence a bit - "The AiiDAlab Quantum ESPRESSO virtual image is based on the  `Quantum Mobile <https://quantum-mobile.readthedocs.io/>`_ image and is available for the following architectures:"

File: docs/source/installation/access_aiidalab/vm.rst [hunk 3]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
 
-#. M1/AMD64 - link coming soon
-#. All other - link coming soon
+Intel x86_64
+------------
+
+Get AiiDAlab Quantum ESPRESSO virtual machine running on your computer in three simple steps:
+
+#. Step1: Download virtual machine image (4.09 GB)
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:16]: It's already a numbered list. No need for "step1", etc.

Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:16]: "Download THE virtual machine...." - same in step 3

File: docs/source/installation/access_aiidalab/vm.rst [hunk 4]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
+
++ username: `max`
++ password: `moritz`
+
+The default configuration of 4 cores and 4096 MB RAM can be adjusted in the VM settings.
+
+M1/M2 apple silicon
+-------------------
+
+If you have an M1/M2 apple silicon, you can use `UTM <https://mac.getutm.app/>`_ to run the AiiDAlab Quantum ESPRESSO virtual machine. The steps are as follows:
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:35]: Should apple and silicon be capitalized? I believe these are proper names of a chip, right? Also, add "chip" after silicon

File: docs/source/installation/access_aiidalab/vm.rst [hunk 5]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
+
+#. Step1: Download compressed virtual machine image (5.9 GB)
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:37]: Same comments as above regarding "step1" and "THE"

File: docs/source/installation/access_aiidalab/vm.rst [hunk 6]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
+
+   + URL: https://bit.ly/486rKtP
+   + Filename: `quantum_mobile_23.10.0-qeapp.utm.zip`
+   + MD5 hash: `44ea9189d788737459c31bf330366926`
+
+#. Step2: Decompress the zip file.
+#. Step3: Install UTM (see https://mac.getutm.app/)
+#. Step4 Import image into UTM by clicking on the Create a New Virtual Machine button and selecting the `quantum_mobile_23.10.0-qeapp.utm` file.
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:45]: Italicize "Create a New Virtual Machine" for emphasis

File: docs/source/installation/access_aiidalab/vm.rst [hunk 7]
```diff
@@ -4,9 +4,49 @@ Virtual Machine Image
...
+Get AiiDAlab Quantum ESPRESSO virtual machine running on your computer in three simple steps:
+
+#. Step1: Download virtual machine image (4.09 GB)
+
+   + URL: https://bit.ly/3Nyf4Dt
+   + Filename: `quantum_mobile_23.10.0-qeapp.ova`
+   + MD5 hash: `5c80d9ab2458f2edac30e97dc2fe36e7`
+
+#. Step2: Install Virtual Box 6.1.6 or later (see https://www.virtualbox.org)
+#. Step3: Import virtual machine image into Virtualbox (11.2 GB) File => Import Appliance
```
Comment (edan-bainglass) [inline docs/source/installation/access_aiidalab/vm.rst:23]: Italicize "Import appliance" for emphasis

**Review by edan-bainglass — CHANGES_REQUESTED** [review]
Please address my comments 🙏

**Review by edan-bainglass — APPROVED** [review]
LGTM!

**Comment by unkcpz** [comment]
Thanks @edan-bainglass

**Pull request merged**